dts/bindings/rtc/nordic,nrf-rtc.yaml

```diff
@@ -11,6 +11,11 @@ properties:
     reg:
       required: true
 
+    cc-num:
+      type: int
+      description: Number of capture/compare (CC) registers available
```

Comment:
only nrf53 has capture capability so maybe just `Number of compare (CC) registers available`

Reply:
Updated, thanks.